templates/components/form/item_device.html.twig

```diff
@@ -109,10 +109,6 @@
                   {% for specificities in specificities_fields %}
                      {% if specificities['canread'] %}
 
-                        {% if specificities['tooltip'] is defined %}
```

Comment:
`specificities['tooltip']` may not be defined. Even if Twig will handle this without any warning, I would prefer to keep the removed test.
To not override `field_options` for all fields, a solution could be to copy it in a new variable:

```
{% set specific_field_options = field_options %}
{% if specificities['tooltip'] is defined %}
    {% set specific_field_options = specific_field_options|merge({'helper': specificities['tooltip']}) %}
{% endif %}
```

Reply:
agree with that